frontend/src/script.ts

```diff
@@ -221,6 +224,7 @@ document.addEventListener('DOMContentLoaded', () => {
     if (!/^https?:\/\//i.test(agentCardUrl)) {
       agentCardUrl = 'http://' + agentCardUrl;
     }
+    const agentCardPath = agentCardPathInput.value.trim() || '/.well-known/agent.json';
```

Comment:
You shouldn't need to do the default backup here, it's already handled on the backend.